doc/getting-started/getting-started.rst

```diff
@@ -608,6 +608,30 @@ Install ACRN
    The target system will reboot into the ACRN hypervisor and
    start the Ubuntu Service VM.
 
+#. Confirm that you see the GRUB menu with the "Ubuntu-ACRN Board Inspector" entry. Select
```

Comment:
Is this new GRUB menu behavior checked in on master or the release_3.2 branch?  I don't see a PR on either branch that would make this change?

Reply:
GRUB menu behavior is checked on release_3.2 branch.